mainloop/game-loop-designs/02-fuggetlen_main_loopok/jatek.cc

```diff
@@ -77,6 +77,7 @@ bool utkozott(sf::Vector2f const& target_pos, sf::Vector2f const& target_size,
 
 
 bool jatek(sf::RenderWindow &window) {
+  // Returns True if player won
```

Comment:
We typically add the comment above the definition of the function. Also, don't write true with a capital, it's quite Python-esque :)